### PR TITLE
feat: Add Call of Duty: Black Ops 3 support

### PR DESCRIPTION
**Pull request by podrivo**
It closes this issue (https://github.com/gamedig/node-gamedig/issues/464).

Also, I'm not sure about the `port_query_offset`, but it worked on the tests I did.

### CODE REVIEW

File: GAMES_LIST.md [hunk 1]
```diff
@@ -61,6 +61,7 @@
 | cod4mw               | Call of Duty 4: Modern Warfare                   |                                             |
 | codenamecure         | Codename CURE                                    | [Valve Protocol](#valve)                    |
 | codenameeagle        | Codename Eagle                                   |                                             |
+| codbo3               | Call of Duty: Call of Duty: Black Ops 3          |                                             |
```
Comment (CosminPerRam) [inline GAMES_LIST.md:64]: Repeated "Call of Duty:"

File: lib/games.js [hunk 2]
```diff
@@ -458,6 +458,14 @@ export const games = {
     },
     release_year: 2007
   },
+  codbo3: {
+    name: 'Call of Duty: Black Ops 3',
+    release_year: 2015,
+    options: {
+      port_query_offset: 2,
```
Comment (CosminPerRam) [inline lib/games.js:465]: If we have a `port_query_offset`, we'd want to also have `port` or `port_query` (to be explicit about from what we are adding/substracting without searching for the default protocol port).
I've found a guide that lists `27017` as default port and all of the 3 servers I found online were on that port, and the query port seems to be the same, so just `port: 27017` is required here.

**Review by CosminPerRam — CHANGES_REQUESTED** [review]
Thank you! Just some small changes please (:

File: GAMES_LIST.md (hunk 1 above)
Comment (podrivo) [reply GAMES_LIST.md:64]: Oops! Sorry about that. Fixed!

File: lib/games.js (hunk 2 above)
Comment (podrivo) [reply lib/games.js:465]: That's good to know! And thank you for founding the default port.
I've copied this from the `Call of Duty: Modern Warfare 3`, so I'll make sure to take a look at as well:

https://github.com/gamedig/node-gamedig/blob/b481425dbf1d50dcb3e756897d7cc15a46cc04c9/lib/games.js#L477-L484

**Review by CosminPerRam — APPROVED** [review]
Thanks!

**Comment by CosminPerRam** [comment]
Oops, seems that this need to be rebased.

**Review by CosminPerRam — APPROVED** [review]

**Pull request merged**